apps/.yarn/patches/@concordium-wallet-connectors-npm-0.6.0-alpha.6-ecfa0628c7.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/dist/WalletConnect.js b/dist/WalletConnect.js
+index 1623aa07e361a157643b80e4c78a243be3539cac..f38017695775d5d2fdec86954d82581989d87617 100644
+--- a/dist/WalletConnect.js
++++ b/dist/WalletConnect.js
+@@ -62,7 +62,7 @@ export const cryptoXWalletTestnet = {
+     id: 'CryptoXTestnet',
+     name: 'CryptoX Wallet (Testnet)',
+     links: {
+-        native: 'cryptoXStage://',
```

Comment:
Do we have a task for fixing this, and maybe include to remove the patch from here